### ISSUE TITLE
Distinct color for hash keys

### ISSUE THREAD

**Issue by nikolay** [atom/language-json#2]
There's almost no syntax highlighting for JSON object. It should make things more readable if the keys had a distinct color.

**Comment by adius** [comment]
+1

**Comment by adius** [comment]
Btw: In newer version of atom the workaround must look like this:

```less
atom-text-editor::shadow .meta.structure.dictionary.json {
  …
}
```

**Comment by 50Wliu** [comment]
Will be available in Atom 1.15.0 as `meta.structure.dictionary.key.json`.
